.github/workflows/ci.yml

```diff
@@ -104,25 +102,29 @@ jobs:
           node-version: 18.15
       - name: Compute node modules cache key
         id: nodeModulesCacheKey
-        run: echo "{value}=$(node build/azure-pipelines/common/computeNodeModulesCacheKey.js)" >> $GITHUB_OUTPUT
+        # {{ SQL Carbon Edit}} Use sql-computeNodeModulesCacheKey
+        run: echo "value=$(node build/azure-pipelines/common/sql-computeNodeModulesCacheKey.js)" >> $GITHUB_OUTPUT
       - name: Cache node modules
         id: cacheNodeModules
         uses: actions/cache@v3
         with:
           path: "**/node_modules"
-          key: ${{ runner.os }}-cacheNodeModules14-${{ steps.nodeModulesCacheKey.outputs.value }}
-          restore-keys: ${{ runner.os }}-cacheNodeModules14-
+          key: ${{ runner.os }}-cacheNodeModules20-${{ steps.nodeModulesCacheKey.outputs.value }}
+          restore-keys: ${{ runner.os }}-cacheNodeModules20-
       - name: Get yarn cache directory path
         id: yarnCacheDirPath
         if: ${{ steps.cacheNodeModules.outputs.cache-hit != 'true' }}
-        run: echo "{dir}=$(yarn cache dir)" >> $GITHUB_OUTPUT
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
       - name: Cache yarn directory
         if: ${{ steps.cacheNodeModules.outputs.cache-hit != 'true' }}
         uses: actions/cache@v3
         with:
           path: ${{ steps.yarnCacheDirPath.outputs.dir }}
           key: ${{ runner.os }}-yarnCacheDir-${{ steps.nodeModulesCacheKey.outputs.value }}
           restore-keys: ${{ runner.os }}-yarnCacheDir-
+      - name: Install libkrb5-dev
```

Comment:
Where is this coming from?

Reply:
Got it from vscode repo ([workflow](https://github.com/microsoft/vscode/blob/033b1e22b783e636e3434f70ca65f2118d22a22a/.github/workflows/monaco-editor.yml#L48)). This is needed since we have intermittent failures for this being missing in workflows ([sample failure](https://github.com/microsoft/azuredatastudio/actions/runs/6579886899/job/17876703787#step:8:67)). so I thought of bringing it over to make workflows more stable.